daemon/containerd/service.go

```diff
@@ -120,10 +119,18 @@ func (i *ImageService) StorageDriver() string {
 	return i.snapshotter
 }
 
-// ReleaseLayer releases a layer allowing it to be removed
+// ReleaseLayer releases the snapshot allowing it to be removed.
```

Comment:
This is confusing; `ReleaseLayer` releasing a snapshot (not a "layer"). How deeply is this function engrained elsewhere? Should we either have separate functions for containerd <--> graphdriver, or would a different (more generic) name  work for the context it's used in?